src/backend/assets/js/session.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function() {
+  var sideslider = $('[data-toggle=collapse-side]');
+  var sel = sideslider.attr('data-target');
+  var top = $('header[role=banner]').outerHeight();
+  var downButton = $('#down-button');
+  var sessionElem = $('.single-session');
+  var sessionId = Number(sessionElem.attr('id'));
+  var maxNoOfSessions = 1000;
```

Comment:
@Princu7 logically, session-id is a unique number that comes from API. So, it's maximum value should be the number of sessions logically.
Now, that length can be calculated on client side.
Also, creating key value pair ( object) dynamically whenever a session is bookmark ed seems reasonable rather than wasting memory in Array. Nice reasoning :)
Yes, make it a separate issue to progress.